elevate-project/constants/routes.js

```diff
@@ -1159,6 +1159,16 @@ module.exports = {
 				path: '/project/v1/users/solutions/:id',
 				type: 'POST'
 			},
+		},
+		{
+			sourceRoute: '/project/v1/users/programs',
+			type: 'GET',
```

Comment:
@Prajwal17Tunerlabs if we are expecting req.body please change the method to POST

Reply:
method changed to POST.